_posts/2020-01-01-SVGD.markdown

```diff
@@ -0,0 +1,405 @@
+feedback: true
+---
+
+This guide is thanks to a many different people, all of whom took their time to give feedback, write reviews, and provide their own insights to the curriculum.
+
+Special thanks to Cinjon Resnick, who patiently waited for months as I procrastinated the completion of this guide, as well as Professor Qiang Liu, who took the time to help shape the curriculum. 
+
+Thank you to Calvin Woo, Sanyam Kapoor, Thomas Pinder, and Swapneel Mehta for useful contributions to this guide, as well as countless insights during our discussions.
+
+A special thanks to the many outside guests who offered to provide their time, including Dilin Wang, Tongzheng Ren, and Haoran Tang.
+
+Finally, thank you to all my fellow students who attended the recitations and provided valuable feedback.
+
+<div class="deps-graph">
+<iframe class="deps" src="/assets/svgd-deps.svg" width="200"></iframe>
+<div>Concepts used in SVGD. Click to navigate.</div>
+</div>
+
+# Why
+
+Stein's Method is a powerful statistical method, one that is at the disposal (and the focus) of many statisticians today. Recently, Stein's Method has made its way into machine learning and has already proved to be a fruitful research area. Stein's Method has deep connections to many machine learning problems of interest, and by the end of this guide, you should be able to understand the relevant mathematics behind this powerful tool.   
+
+<br />
+
+# 1 Basics Behind Kernelized Stein Discrepancy
+  **Motivation**: Before jumping into all the math and methodology, we have to be able to  understand the basics of what’s going on. Most importantly, we will review the basics of measure theory and reproducing kernel hilbert spaces. Measure theory allows us to understand the notion of discrepancy measures between distributions, which we will use later on to quantify the difference between two arbitrary distributions of interest. Our other topic, Reproducing Kernel Hilbert Spaces (RKHS), will serve as the connection between measure theory and a practical machine learning algorithm. With RKHS, we will be able to define and optimize intractable measures which previously, were only useful for theoretical analysis or a restrictive class of functions. These two together set the foundation for defining a tractable Kernelized Stein Discrepancy, which serves as the driving factor behind Stein Variational Gradient Descent. 
+
+
+  **Topics**:
+
+  1. Measure Theory
+  2. Kernels
+  3. Reproducing Kernel Hilbert Space
+  4. Machine Learning Basics
+
+  **Notes**: In this class, with notes in [Colab](https://colab.research.google.com/drive/1x3bgKtYWaYRTV1VGaf0bKRyQ_qxNZpjh) or [PDF](/assets/svgd_notes/week01.pdf), we went over the basic mathematical concepts we will need throughout the rest of the curriculum.
+
+  **Required Reading**:
+
+  1. [Reproducing Kernel Hilbert Spaces Tutorial, Section 1 - 3](http://www.gatsby.ucl.ac.uk/~gretton/coursefiles/lecture4_introToRKHS.pdf)
```

Comment:
Every reading should have a period after it, e.g. '[Reproducing Kernel Hilbert Spaces Tutorial, Section 1 - 3].'